<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>972</width>
    <height>570</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="palette">
    <palette>
     <active/>
     <inactive/>
     <disabled/>
    </palette>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>951</width>
      <height>531</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QGroupBox" name="top_groupBox">
       <property name="title">
        <string/>
       </property>
       <widget class="QGroupBox" name="contrl_groupBox">
        <property name="geometry">
         <rect>
          <x>680</x>
          <y>0</y>
          <width>261</width>
          <height>261</height>
         </rect>
        </property>
        <property name="title">
         <string/>
        </property>
        <layout class="QGridLayout" name="gridLayout">
         <item row="2" column="0">
          <widget class="QPushButton" name="pushButton_6">
           <property name="text">
            <string>PushButton</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QPushButton" name="pushButton_5">
           <property name="text">
            <string>PushButton</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QPushButton" name="pushButton_3">
           <property name="text">
            <string>PushButton</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QPushButton" name="btnStop">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>停止录制</string>
           </property>
           <property name="icon">
            <iconset resource="img.qrc">
             <normaloff>:/icon/C:/Users/Liao/Downloads/stop.png</normaloff>:/icon/C:/Users/Liao/Downloads/stop.png</iconset>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QPushButton" name="btnStart">
           <property name="text">
            <string>开始录制</string>
           </property>
           <property name="icon">
            <iconset resource="img.qrc">
             <normaloff>:/icon/C:/Users/Liao/Downloads/start.png</normaloff>:/icon/C:/Users/Liao/Downloads/start.png</iconset>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QPushButton" name="btnPlayHistory">
           <property name="text">
            <string>查看历史回放</string>
           </property>
           <property name="icon">
            <iconset theme="accessories-calculator"/>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="thum_groupBox">
       <property name="title">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="img.qrc"/>
 </resources>
 <connections/>
</ui>
